### PR TITLE
Setup test environment

### PR DESCRIPTION
**Pull request by liondon**

### CODE REVIEW

File: service/__init__.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+"""
```
Comment (nspektor) [inline service/__init__.py:1]: isnt this file also in your other PR? we dont want to have a merge conflict

File: service/service.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
+"""
```
Comment (nspektor) [inline service/service.py:1]: same here

**Review by nspektor — CHANGES_REQUESTED** [review]
lets make sure we dont have repeated changes between PRs

File: service/__init__.py (hunk 1 above)
Comment (liondon) [reply service/__init__.py:1]: You're right. I kept working on this story after I finished the last one and didn't start from where the origin/master was

**Review by nspektor — APPROVED** [review]

**Pull request merged**